pennylane/beta/queuing/measure.py

```diff
@@ -0,0 +1,205 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# pylint: disable=protected-access
+"""
+This module contains the functions for computing different types of measurement
+outcomes from quantum observables - expectation values, variances of expectations,
+and measurement samples using AnnotatedQueues.
+"""
+import pennylane as qml
+from pennylane.qnodes import QuantumFunctionError
+
+# --------------------
+# Beta related imports
+# --------------------
+from pennylane.beta.queuing.operation import Observable, Sample, Variance, Expectation, Probability, Tensor
+from pennylane.beta.queuing.ops import Identity
```

Comment:
Most of the related files would turn to imports from the "beta" modules rather than for example the `qml.Identity` shorthand